watchman/checks.py

```diff
@@ -63,7 +63,8 @@ def _check_storage():
     path = default_storage.save(filename, ContentFile(content))
     default_storage.size(path)
     default_storage.open(path).read()
-    default_storage.delete(path)
```

Comment:
Would we want to have the same file name each time we save? It looks like it uses uuid4 for the file name.

Reply:
I think that's probably intentional right now, so that you don't keep overwriting the same file each time the check is performed.

Our storage backend is configured as S3, and the app we're using django-watchman on doesn't have the S3 Delete permission on that file path. We're thinking of setting a rule on the bucket to auto-expire the files created by django-watchman automatically after a day. That'll keep em cleaned up, without having to give Delete permission to the app